### DISCUSSION TITLE
End-of-life policy

### DISCUSSION THREAD

**Discussion by mortenjoenby** [keycloak/keycloak#19937] [Q&A]
### Description

Is my understanding correct, that there is no end-of-life policy on any Keycloak versions?

**Comment by jonkoops** [comment]
We do not offer support on any version of Keycloak, the latest version is the only actively supported version. Therefore any version that is not the latest version is technically end-of-life.

If you are looking for a commercially supported version of Keycloak we offer [Red Hat Single Sign-On](https://access.redhat.com/products/red-hat-single-sign-on) for that.
